### DISCUSSION TITLE
Breaking change: Microsoft Teams

### DISCUSSION THREAD

**Discussion by aaronparker** [EUCPilots/evergreen-module#645] [General]
I've pushed a change to the current `development` branch that renames the Microsoft Teams functions. This will require a change to scripts that uses these functions, after the next Evergreen release:

* `MicrosoftTeams` to `MicrosoftTeamsClassic`
* `MicrosoftTeamsPreview` to `MicrosoftTeams`
* Updates the app name for each production

**Comment by xenadmin** [comment]
What's the best channel to subscribe, so I can adjust my scripts on that day? This discussion here?

**Comment by aaronparker** [comment]
I guess so, but I'll have to remember to post an update. I'll try to select a date/time to push the new version before pushing to the PowerShell gallery.

Otherwise hard code your scripts for the current version of the module.

**Comment by aaronparker** [comment]
I'm ready to release this next week, so will look at doing on the Apr 9th AEST (Apr 8th for those in US, early 9th for the EU etc.)

**Comment by xenadmin** [comment]
Thank you very much for your reminder. We will create new Master Images on Patch Tuesday +1, so this will be 10th April in the EU.
Therefore, I will adjust our Scripts to use the expression `MicrosoftTeamsClassic` starting tomorrow.

**Comment by aaronparker** [comment]
This has been pushed to the PowerShell gallery

**Comment by xenadmin** [comment]
Thank you very much. I can confirm that my query works as expected:
`Get-EvergreenApp -Name MicrosoftTeamsClassic | Where-Object { $_.Architecture -eq "x64" -and $_.Ring -like "General" -and $_.Type -eq "msi" }`

**Comment by chezzer64** [comment]
Just a thought - it it worth mentioning this breaking change in World of EUC Slack forum - as I suspect quite a lot of people that use Evergreen are more likely to see it there?
